packages/website/ts/components/onboarding/portal_onboarding_flow.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Step } from 'react-joyride';
+
+import { OnboardingFlow } from 'ts/components/onboarding/onboarding_flow';
+
+export interface PortalOnboardingFlowProps {
+    stepIndex: number;
+    isRunning: boolean;
+    setOnboardingShowing: (isShowing: boolean) => void;
+}
+
+const steps: Step[] = [
+    {
+        target: '.wallet',
+        content: 'You are onboarding right now!',
+        placement: 'right',
+        disableBeacon: true,
+    },
+];
+
+export class PortalOnboardingFlow extends React.Component<PortalOnboardingFlowProps> {
```

Comment:
I expect there to be more helper functions I'll be needing and I like the class-syntax for that instead of defining the functions in-line. If the component is still this simple when we're done with the onboarding flow I'll convert it to a stateless component. 
I've always been a bit confused about stateless components. They're not more efficient than stateful components at the moment, they just provide a nicer syntax. I think the the only argument for using them is that they enforce better conventions (smart vs. dumb components etc...)